src/flexbox/nasa.css

```diff
@@ -0,0 +1,133 @@
+body{
+  display:flex;
+  flex-direction: column;
+  height: 800px;
+  width: 600px;
+  
```

Comment:
i'm not a big fan of setting a specific height/width on body. these should be 100% you should wrap the <div class="image"> and <div class="footer"> elements with a <div class="main-container">. and set the width and height on that.